snippets.json

```diff
@@ -0,0 +1,53 @@
+// cSpell:ignore mdlint
+{
+	"insertMarkdownLintDisableComment": {
+		"prefix": [
+			"markdownlint-disable",
+			"mdlint-disable",  // For convenience.
+		],
+		"body": "<!-- markdownlint-disable -->",
+		"description": "Disables all markdownlint rules unconditionally. Takes effect starting with the line this comment is on."
+	},
+	"insertMarkdownLintEnableComment": {
+		"prefix": [
+			"markdownlint-enable",
+			"mdlint-enable",  // For convenience.
+		],
+		"body": "<!-- markdownlint-enable -->",
+		"description": "Enables all markdownlint rules unconditionally. Takes effect starting with the line this comment is on."
+	},
+	
+	"insertMarkdownLintDisableRulesComment": {
+		"prefix": [
+			"markdownlint-disable-rules",
+			"mdlint-disable-rules",  // For convenience.
+		],
+		"body": "<!-- markdownlint-disable ${1:$TM_SELECTED_TEXT} -->",
```

Comment:
Do you think that the additional cognitive load of overloading the description of a single snippet with two different use cases and associated syntaxes is justified by the smaller number of snippets? (serious question) If so, then I will go ahead and make the change for both `markdownlint-disable` and `markdownlint-enable`.

Reply:
In general, I prefer less code to more code. That said, I don’t know what the experience is like as you have it now versus how it might be as I propose. I defer to your experience and opinion.

That said, if I try it later and feel that it is very cumbersome, I may try the other way to judge for myself. :)

Reply:
On second thought, I decided to remove the separate `*-rules` snippets and just add a tab stop plus more documentation to the core ones. 41ca6f5